tools/wptrunner/wptrunner/browsers/chrome_android.py

```diff
@@ -48,18 +48,20 @@ def executor_kwargs(test_type, server_config, cache_manager, run_info_data,
                                            **kwargs)
     executor_kwargs["close_after_done"] = True
     capabilities = dict(DesiredCapabilities.CHROME.items())
-    capabilities["chromeOptions"] = {}
-    # required to start on mobile
-    capabilities["chromeOptions"]["androidPackage"] = "com.google.android.apps.chrome"
+    capabilities["goog:chromeOptions"] = {}
+    # TODO(chrome): browser_channel should be properly supported.
+    package_name = "com.android.chrome"  # stable channel
+    # Required to start on mobile
+    capabilities["goog:chromeOptions"]["androidPackage"] = package_name
+    # Note: from ChromeDriver 75 and onwards only w3c is supported.
+    capabilities["goog:chromeOptions"]["w3c"] = True
```

Comment:
I think this is the default now and there's no need to specify it?

Reply:
Removed.